adapters/handlers/rest/configure_api.go

```diff
@@ -696,12 +696,47 @@ func registerModules(appState *state.State) error {
 
 	appState.Modules = modules.NewProvider()
 
+	// Default modules
+	defaultVectorizers := []string{
+		"text2vec-aws",
+		"text2vec-cohere",
+		"text2vec-huggingface",
+		"text2vec-jina",
+		"text2vec-octo",
+		"text2vec-openai",
+		"text2vec-palm",
+		"text2vec-voyageai",
+	}
+	defaultGenerative := []string{
+		"generative-anyscale",
+		"generative-aws",
+		"generative-cohere",
+		"generative-mistral",
+		"generative-octoai",
+		"generative-openai",
+		"generative-palm",
+	}
+	defaultOthers := []string{
+		"reranker-cohere",
+		"reranker-voyageai",
+	}
```

Comment:
Please use `.Name` constants to get the name of the module (eg. `modgenerativemistral.Name`) to mitigate the misspellings.

Reply:
Updated